### PR TITLE
Updated ACS Rooms CLI to reference 1.0.0b3 SDK

### PR DESCRIPTION
**Pull request by mikehang-msft**
Updated ACS Rooms CLI to reference 1.0.0b2 SDK
Updated Rooms tests and Recorded test files

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/communication/azext_communication/manual/_help.py [hunk 1]
```diff
@@ -464,6 +461,15 @@
                az communication rooms delete --room "12345678901234567"
 """
 
+helps['communication rooms list'] = """
+    type: command
+    short-summary: "List all active rooms belonged to a current Communication Service resource."
```
Comment (0rland0Wats0n) [inline src/communication/azext_communication/manual/_help.py:466]:
```suggestion
    short-summary: "List all active rooms belonging to a current Communication Service resource."
```

File: src/communication/azext_communication/version.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # --------------------------------------------------------------------------------------------
 
 
-VERSION = '1.5.2'
+VERSION = '1.5.3'
```
Comment (mayssamm) [inline src/communication/azext_communication/version.py:7]: I think the changes are not that minor, I recommend bumping to 1.6

**Comment by mayssamm** [comment]
Please update the history file as well (communication/HISTORY.rst).

**Comment by mayssamm** [comment]
Also, I believe you need to require a minimum version for the SDK as these changes do not work with previous versions of the rooms python sdk.

File: src/communication/azext_communication/version.py (hunk 2 above)
Comment (mikehang-msft) [reply src/communication/azext_communication/version.py]: bumped to 1.6

**Review by mayssamm — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

File: src/communication/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.6
```
Comment (kairu-ms) [inline src/communication/HISTORY.rst:6]: Please use `1.6.0`

File: src/communication/azext_communication/version.py [hunk 4]
```diff
@@ -4,7 +4,7 @@
 # --------------------------------------------------------------------------------------------
 
 
-VERSION = '1.5.2'
+VERSION = '1.6'
```
Comment (kairu-ms) [inline src/communication/azext_communication/version.py:7]: Please use 1.6.0

File: src/communication/HISTORY.rst (hunk 3 above)
Comment (mikehang-msft) [reply src/communication/HISTORY.rst:6]: updated

**Pull request merged**